test/cpp/jit/test_alias_analysis.h

```diff
@@ -11,6 +11,12 @@
 namespace torch {
 namespace jit {
 
+inline c10::OperatorOptions aliasAnalysisFromSchema() {
```

Comment:
This pattern appears many times in this PR; seems like a sign that the API is clunky. Are users given a different API?

Reply:
This is clunky because `Operator` objects are instantiated manually. If you use the public API (i.e. `torch::RegisterOperators`), it is far less clunky.